Workbooks/AKS/Node Kubelet AKS/Node Kubelet AKS.workbook

```diff
@@ -5,7 +5,46 @@
       "type": 9,
       "content": {
         "version": "KqlParameterItem/1.0",
+        "crossComponentResources": [
+          "{Resource}"
+        ],
         "parameters": [
+          {
+            "id": "f90f6fdf-8238-496c-ac1e-90ec2883de6f",
+            "version": "KqlParameterItem/1.0",
+            "name": "Resource",
+            "type": 5,
+            "isRequired": true,
+            "value": "value::1",
+            "isHiddenWhenLocked": true,
+            "typeSettings": {
+              "resourceTypeFilter": {
+                "microsoft.containerservice/openshiftmanagedclusters": true,
```

Comment:
If you are adding support for ARO v3 and v4 then you should include these everywhere

ARO v3 => 'Microsoft.ContainerService/openshiftManagedClusters'
ARO v4 => 'Microsoft.RedHatOpenShift/OpenShiftClusters'

AKS => 'Microsoft.ContainerService/managedClusters' 
Azure Arc K8s => 'Microsoft.Kubernetes/connectedClusters'